Library/Formula/liblqr.rb

```diff
@@ -3,7 +3,7 @@
 class Liblqr < Formula
   desc "C/C++ seam carving library"
   homepage 'http://liblqr.wikidot.com/'
```

Comment:
This one can be updated to `https://`, which will redirect to plaintext (for non-logged-in users I guess).